mypyc/genops.py

```diff
@@ -99,11 +99,30 @@ def f(x: int) -> int:
 GenFunc = Callable[[], None]
 
 
+class UnsupportedException(Exception):
+    pass
+
+
+class Errors:
+    def __init__(self) -> None:
+        self.num_errors = 0
+        self.num_warnings = 0
+
+    def error(self, msg: str, path: str, line: int) -> None:
+        print('{}:{}: error: {}'.format(path, line, msg))
```

Comment:
Currently errors are generated in somewhat arbitrary order. What about sorting the errors by file and line number?